### PR TITLE
RATEST-102: Merge qa branch with master

### PR DESCRIPTION
**Pull request by k-joseph**

### CODE REVIEW

File: ui-tests/src/test/java/org/openmrs/reference/page/AdministrationPage.java [hunk 1]
```diff
@@ -16,7 +16,7 @@
 
     private static final String URL = "/admin/index.htm";
 
-    private static final By MANAGE_USERS = By.cssSelector("#legacyui-manageUsers a");
+    private static final By MANAGE_USERS = By.cssSelector("#legacyui-mangeUsers a");
```
[the remark this answers is not in the document]
Comment (sherrif10) [reply ui-tests/src/test/java/org/openmrs/reference/page/AdministrationPage.java:19]: Sure i had commited this on legacy ui ,however let me commit this fix quickly

[the remark this answers is not in the document]
Comment (k-joseph) [reply ui-tests/src/test/java/org/openmrs/reference/page/AdministrationPage.java:19]: remember you will need to update this in qa here, and change legacy version too

[the remark this answers is not in the document]
Comment (k-joseph) [reply ui-tests/src/test/java/org/openmrs/reference/page/AdministrationPage.java:19]: you can attached https://github.com/openmrs/openmrs-distro-referenceapplication/pull/324#discussion_r591113967 onto the ticket and the ticket here for later reference

[the remark this answers is not in the document]
Comment (sherrif10) [reply ui-tests/src/test/java/org/openmrs/reference/page/AdministrationPage.java:19]: Ticketed it here https://github.com/openmrs/openmrs-distro-referenceapplication/pull/325 ,Let me update the legacyui version

[the remark this answers is not in the document]
Comment (sherrif10) [reply ui-tests/src/test/java/org/openmrs/reference/page/AdministrationPage.java:19]: @k-joseph  did you close this  the ticket

[the remark this answers is not in the document]
Comment (sherrif10) [reply ui-tests/src/test/java/org/openmrs/reference/page/AdministrationPage.java:19]: ohh sorry i saw your merge ok thanks

**Pull request merged**